pyof/v0x01/common/flow_match.py

```diff
@@ -119,3 +120,27 @@ def __init__(self, wildcards=None, in_port=None, dl_src=None, dl_dst=None,
         self.nw_dst = nw_dst
         self.tp_src = tp_src
         self.tp_dst = tp_dst
+
+        if self.wildcards is None:
+            self.fill_wildcards()
+
+    def fill_wildcards(self):
+        """Fill wildcards considering the attributes of the instance."""
```

Comment:
Please @macartur , try to describe the method structure/algorithm on its description, so we know the logic inside without the need to read its code.

There are other methods on our source code that need this kind of improvement - we are discovering it after trying to use some methods weeks/months after creating them.